step_definitions/tasks/deleteTask_steps.js

```diff
@@ -6,4 +6,14 @@ const TASKListPage = require('../../page-objects/pages/TASKListPage.js');
 
 When(/^.* clicks on Delete task button$/, async function () {
 	await TASKListPage.clickDeleteTaskButtonInPopup();
-});
\ No newline at end of file
+});
+
+Then(/^.* should not see task with name '([^']*)' visible$/, async function (taskName) {
```

Comment:
Don't you think, it makes sense to use this on all tests, that use that step, instead of creating a whole new one?